abixen-platform-business-intelligence-service/src/main/java/com/abixen/platform/service/businessintelligence/multivisualization/form/DatabaseConnectionForm.java

```diff
@@ -20,34 +20,59 @@
 import com.abixen.platform.service.businessintelligence.multivisualization.model.impl.database.DatabaseConnection;
 import com.fasterxml.jackson.annotation.JsonView;
 
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
 
 public class DatabaseConnectionForm implements Form {
 
+    private static final int NAME_MAX_LENGTH = 40;
```

Comment:
Move it to the com.abixen.platform.service.businessintelligence.multivisualization.model.impl.database.DatabaseConnection 
and use bean validator in this file as well.
Add changes to liquibase file as well. 
Make DATABASE_HOST_MAX_LENGTH = 255;
DESCRIPTION_MAX_LENGTH = 1000;